demo/src/router.ts

```diff
@@ -1,10 +1,13 @@
 import Menu from "./Menu.svelte";
 
 export const examples = Object.entries(import.meta.globEager("./examples/**.svelte")).map(([path, component]) => {
-  const name = path.match(/\/\d+\s([^/]+)\./)?.[1].toString() ?? "";
+  const parsedFileName = path.match(/\/(\d+)\s([^/]+)\./);
+  const index = parseInt(parsedFileName[1]);
```

Comment:
Saving the example's index to be able to sort by it later (when displaying in the menu).

Reply:
OK